### DISCUSSION TITLE
How to set up each fuzz in parallel testing?

### DISCUSSION THREAD

**Discussion by chennbnbnb** [AFLplusplus/AFLplusplus#1817] [General]
I was fuzzing on a machine with 16 cores, and I encountered a problem that required experience
1. Should I use hyperthreading? Should I enable 16 or 32 fuzzers
2. How to combine the parameters of -l in cmplog?
3. How to choose the -L parameter?
4. How to combine the -L parameter with the -p or -Z parameter?
These are not mentioned in the document, I hope to know some practical experience, thank you

**Comment by vanhauser-thc** [comment]
Do you find any answers missing after reading the multicore section in the fuzzing_in_depth document?
